src/paymentservice/charge.js

```diff
@@ -1,24 +1,39 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
-const {context, propagation, trace, metrics} = require('@opentelemetry/api');
+const { context, propagation, trace, metrics } = require('@opentelemetry/api');
 const cardValidator = require('simple-card-validator');
 const { v4: uuidv4 } = require('uuid');
 
 const { OpenFeature } = require('@openfeature/server-sdk');
-const { FlagdProvider} = require('@openfeature/flagd-provider');
+const { FlagdProvider } = require('@openfeature/flagd-provider');
 const flagProvider = new FlagdProvider();
 
 const logger = require('./logger');
 const tracer = trace.getTracer('paymentservice');
 const meter = metrics.getMeter('paymentservice');
-const transactionsCounter = meter.createCounter('app.payment.transactions')
+const transactionsCounter = meter.createCounter('app.payment.transactions');
+
+const TENANT_LEVEL = ['gold', 'silver', 'bronze'];
```

Comment:
By tenant level you mean something like user level?
Tenant is a bit misguiding to me.

Reply:
It's more of a customer status level, like a rewards programme level highlighting their customer loyalty. So, this could be change to `status_level` or `loyalty_level`.

Reply:
+1 for `loyalty_level`